<launch>

  <!--name argument is not required, because it will overwrite all node name in the exec-->
  <node pkg="lego_loam_bor" exec="lego_loam" output="screen" respawn="false">
    <param from="$(find-pkg-share lego_loam_bor)/config/loam_c16_config.yaml" />
    <remap from="/lslidar_point_cloud" to="/lslidar_point_cloud"/>
  </node>

  <node pkg="rviz2" exec="rviz2" output="screen" respawn="false" args="-d $(find-pkg-share lego_loam_bor)/rviz/lego_loam.rviz">
  </node>

  <!--- TF -->
  <node pkg="tf2_ros" exec="static_transform_publisher" name="camera_init_to_map"  args="0 0 0 1.570795 0 1.570795 map camera_init" />
  <node pkg="tf2_ros" exec="static_transform_publisher" name="sensor2baselink"  args="0.3 0 -0.3 3.1415926535 0 0 laser_link base_link" />
  
</launch>
